package.json

```diff
@@ -29,10 +44,10 @@
     "test": "npm run compile && node ./node_modules/vscode/bin/test"
   },
   "devDependencies": {
+    "@types/mocha": "^2.2.42",
```

Comment:
I removed the tests, so I should also remove mocha.

(or actually add tests, probably the better route)